### DISCUSSION TITLE
Projects new: "Done at", "Created at" fields groupable by date

### DISCUSSION THREAD

**Discussion by iliubinskii** [community/community#12387] [Projects and Issues]
It would be nice to have "Done at", "Created at" fields groupable by date.

This would let see progress (i.e. the list of done tasks) for each day.

**Comment by rayalan** [comment]
I was looking to do something similar as a recap of work completed. I can filter only to done work, but I can't indicate when it was done without a "Done At" or similar field.

**Comment by Raisolution** [comment]
We are aiming to achieve similar functionality. Basically - have the information of `when` did an item changed its status (in progress, done etc.)
